app/views/publish/courses/requirements/edit.html.erb

```diff
@@ -0,0 +1,58 @@
+<% if @provider.recruitment_cycle_year.to_i >= Provider::CHANGES_INTRODUCED_IN_2022_CYCLE %>
+  <% page_title = "Personal qualities and other requirements" %>
+<% else %>
+  <% page_title = "Requirements and eligibility" %>
+<% end %>
+
+<% content_for :page_title, title_with_error_prefix("#{page_title} – #{@course.name_and_code}", @course_requirement_form.errors.any?) %>
+
+<% content_for :before_content do %>
+  <%= govuk_back_link_to(publish_provider_recruitment_cycle_course_path(@provider.provider_code, @course.recruitment_cycle_year, @course.course_code)) %>
+<% end %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= form_with(
+      model: @course_requirement_form,
+      url: requirements_publish_provider_recruitment_cycle_course_path(@provider.provider_code, @course.recruitment_cycle_year, @course.course_code),
+      data: { qa: "enrichment-form", module: "form-check-leave" },
+      method: :patch,
+      local: true,
+    ) do |f| %>
+
+      <%= f.govuk_error_summary %>
+
+      <h1 class="govuk-heading-l">
+        <span class="govuk-caption-l"><%= @course.name_and_code %></span>
+        <%= page_title %>
+      </h1>
+
+      <% if @provider.recruitment_cycle_year.to_i < Provider::CHANGES_INTRODUCED_IN_2022_CYCLE %>
```

Comment:
@defong Do we still need to deal with this?

Reply:
That can be drop, tech debt that was incurred due to last rollover changes